taskfile/task.go

```diff
@@ -36,6 +35,19 @@ type Task struct {
 	IncludeVars          *Vars
 	IncludedTaskfileVars *Vars
 	IncludedTaskfile     *IncludedTaskfile
+	Platforms            []*Platform
+}
+
+func ShouldRunOnCurrentPlatform(platforms []*Platform) bool {
+	if len(platforms) == 0 {
+		return true
+	}
+	for _, platform := range platforms {
+		if platform.MatchesCurrentPlatform() {
+			return true
+		}
+	}
+	return false
```

Comment:
This function feels a little lonely now that all the other platform related changes have moved. Maybe it would be better in [task.go](https://github.com/go-task/task/blob/master/task.go) alongside the Executor which is where its called from. The `taskfile` package is generally for code that reads/parses the Taskfile` whereas this is called during the execution of a command.

Reply:
Good call. Pushed change 👍